data/coalition/coalition missions.txt

```diff
@@ -1393,3 +1393,65 @@ mission "Arachi Cartoonist 2"
+			`	While waiting, you look back at the Alpha to find his eyes fixated on you, not looking away or even blinking. After you stare him down for a couple of seconds, he speaks up. "What is it, are you expecting some sort of apology? I'd consider my caution to be entirely justified, so you won't get one." After a long pause, he continues, "I'm sure you have questions though. I've given up on fighting humanity left and right, and it seems that I won't have anything better to do for some time. Ask away."`
+			choice
+				`	"Who are you, exactly? How did you even get here?"`
+				`	"What are you doing here in the Coalition?"`
+			`	"Where do I begin? Well, for starters, name's Alitis. Was Athis before, but I took a Saryd name after I got here. Back in the 2800s, the crew I was on found some aliens somewhere far north of Earth. We didn't get much from them though, as they were fighting another group of aliens, ones who had jump drives. We didn't want any part of the fighting because the battle wasn't favorable to us, but we managed to loot a jump drive from the second group of aliens. After that, we started exploring disconnected systems to the east. Eventually, we joined an enclave in the Dirt Belt, but it didn't take long for the Navy to find us there. While evacuating, some of the others wanted to head back east, but I knew better, so I banked on heading further west, and after some time got here."`
+			choice
+				`	"You 'knew better?' What do you mean?"`
+				`	"What, did you have a change of heart once you got here?"`
+			`	He sits on the ground for a while, thinking. "I was one of those that fought humanity after the first Turf War. I was ambushed, cornered many times - fear wasn't a stranger to me. After we stole that jump drive and ventured east, however, someone or... something rather, some voice spoke to me. It was the first time I felt scared. After that, I wasn't inclined to go back east with the others, no matter how silly a reason they thought it was. When I got here, me and my crew helped the Heliarchs with translations for humans, and I told them about humanity and the Alphas. They took me in, but demanded my jump drive, so I handed it over. Damn thing got me in more trouble than it was worth anyway."`
+			choice
+				`	"So where are the other Alphas that came with you?"`
+					goto others
+				`	"What did the voice say?"`
+			`	He forces a chuckle out. "You got here, so you have a jump drive, I assume? Go on, head east and maybe it'll speak to you too. Find out if it classifies you as 'disposable' as well."`
+				goto end
+			label others
+			`	"Others? Oh, you misunderstood me. I took a crew of brigands and pirates. They're all long dead now, most of old age. I'd say they made the right choice in coming with me, no matter how much they protested that the aliens here were too... alien."`
+			label end
```

Comment:
Changing the choice since I removed the mention of other crew, and also expanding upon the responses a bit.
```suggestion
				`	"Voices in your head? Are you sure you weren't just imagining them?"`
					goto imaginary
				`	"What did the voice say?"`
			`	He looks away from you. "You're here now, aren't you? This means you must have a device to jump beyond links as I did. Travel to the east and find out for yourself what the voices say. Find out if it classifies you as 'disposable' as well.`
				goto end
			label imaginary
			`	He looks away from you. "No imaginary thing could have such an impact on me. You're here now, aren't you? This means you must have a device to jump beyond links as I did. Travel to the east and find out for yourself that the voices are real."`
			label end
```